packages/hooks/src/useToggle/index.ts

```diff
@@ -1,50 +1,48 @@
-import { useState, useMemo } from 'react';
+import { useMemo, useState } from 'react';
 
-type IState = string | number | boolean | undefined;
-
-export interface Actions<T = IState> {
+export interface Actions<T> {
   setLeft: () => void;
   setRight: () => void;
-  toggle: (value?: T) => void;
+  set: (value?: T) => void;
```

Comment:
`set` 函数的 `value` 参数不能缺省吧……毕竟都已经和 `toggle` 分开了
```suggestion
  set: (value: T) => void;
```